sdl2promela/stop_condition/translator.py

```diff
@@ -424,6 +462,22 @@ def _find_first_variable(
 @dispatch(GenerateContext, model.Selector)
 def _generate(context: GenerateContext, expr: model.Selector):
 
+    if context.process_state_selection is not None:
+        # a special case
```

Comment:
Please elaborate why this is special?

Reply:
the comment was removed, the function with proper name was extracted